classes/controller/AdminController.php

```diff
@@ -3307,7 +3307,7 @@ public function getModulesList($filter_modules_list)
             $filter_modules_list = array($filter_modules_list);
         }
 
-        if (is_null($filter_modules_list) || !count($filter_modules_list)) {
+        if (is_null($filter_modules_list) || !is_array($filter_modules_list)) {
```

Comment:
Should use `empty`,  `is_array` checked above and here it will allow empty array to pass through